spec/requests/custom_actions_spec.rb

```diff
@@ -77,6 +77,15 @@ def expect_result_to_have_custom_actions_hash
       expect_result_to_have_keys(%w(id href actions))
       expect(response.parsed_body["actions"].select { |a| a["method"] == "post" }.pluck("name")).to match_array(%w(edit add_resource remove_resource remove_all_resources add_provider_vms))
     end
+
+    it "allows expanding of custom actions" do
+      FactoryGirl.create(:service)
```

Comment:
Move as a memoized helper after line 70 (`let(:service)....`), this is test setup that can be reused in this block and kept out of the spec itself.